Adds CC Security checkpoints to Central Commands

CawsForConcern opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Adds pre-built security checkpoints to Central Commands SC17 and G24.

**These checkpoints are open by default,** but can be **closed by admins/eventmins** and generally be operated by CC staff. 

Some features include:
- A Security Records computer inside the checkpoint to reference.
- An empty table that admins can fill with stuff like Ambuzol Plus.
- A mailing unit that is hooked up straight to the Armory in order to deposit contraband into.
- Checkpoint 'status' lights. A green light by the exit turns on when the exit door is opened, and a red light when it's closed.

Also included in this PR:
- Screens placed around both of the Central Commands, especially to places where you wanted to be able to see the end-of-round timer like Command Debrief, the Bar, Medical, Security, etc.
- More rechargers in departments and public areas. Lets people recharge their translators and various other gadgets.
- Added a sink to G24's kitchen
- Fixes a disconnected APC by G24's armory

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
The intention of this PR is just to make running checkpoints easier and more immersive. The setups are meant to be very simple to operate (one button closes the exit, one button closes the entry) with minimal fuss. They're also built to be reasonably secure so you don't get random people walking into the control-sides and letting everyone through all in one go.

The checkpoints are built to be "open by default," so if there's no admins online or no event going on, nothing has to be done to allow people through them. The doors and buttons are set up in such a way that the checkpoint has to be closed manually by an admin or by Central Command staff.

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->
<img width="1146" height="1001" alt="image" src="https://github.com/user-attachments/assets/fef96920-9dbf-4011-bac1-075d35491ca9" />

fig. 1 - Checkpoint at Central Command G24

<img width="1285" height="998" alt="image" src="https://github.com/user-attachments/assets/2f06bd46-8424-464b-97b2-94f1df4c4cbf" />

fig. 2 - Checkpoint at Central Command SC17

<img width="1146" height="1001" alt="image" src="https://github.com/user-attachments/assets/7a9b4a74-a3f1-4c59-aecf-dbf90be6d971" />

fig. 3 - Explanation of what the buttons do (they're labeled in-game as well so you can right-click examine them as well)

## Checks
<!-- check boxes for faster reviewing of your PR -->

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->
:cl: CawsForConcern
- add: (Centcomm G24) CC Security checkpoint between Arrivals and Lobby
- add: (Centcomm G24) Sink to kitchen
- add: (Centcomm G24) Added more rechargers, useful for charging your translators and other gadgets
- fix: (Centcomm G24) Disconnected APC by Armory
- add: (Centcomm SC17) CC Security checkpoint between Arrivals and Lobby
- add: (Centcomm SC17) Added more rechargers, useful for charging your translators and other gadgets
- add: (Centcomm SC17) Two more mindshields in CC Debrief

CawsForConcern committed:
- add: (Centcomm G24) Security / Customs checkpoint that can be used by admins for events

CawsForConcern committed:
- add: (Centcomm SC17) Security / Customs checkpoint that can be used by admins for events
- tweak: (Centcomm G24) Wiring adjustments

CawsForConcern committed:
- add: (Centcomm SC17)  Screens (to show end-of-round countdown)
- add: (Centcomm G24) Screens (to show end-of-round countdown)

CawsForConcern committed:
electrification tweaks

CawsForConcern committed:
button labels updated for exit/entry switches

CawsForConcern committed:
One more wiring consistency tweak

CawsForConcern commented:
Draft until I get the green light on the layout / aesthetic matching on Centcomm SC17

Littlemen07 commented:
i like the idea and i think player culture will slowly addapt to it, however i think this is very small for the ~120 pop of alpha

PubliclyExecutedPig commented:
Looks fine to me, are you waiting for someone else's green light?

CawsForConcern commented:
> Looks fine to me, are you waiting for someone else's green light?

Trep wanted to have input on SC17, was waiting on their reply before un-drafting this

CawsForConcern commented:
> i like the idea and i think player culture will slowly addapt to it, however i think this is very small for the ~120 pop of alpha

I'm a little stuck between "don't make it so large that it increases walking time into CC substantially" and designing it so it can handle a fair number of people. At the point the checkpoint is roughly bigger than one screen, I think you'll just run into people not realizing that one of the line-ups is shorter, or making it hard for people playing CC ghost roles to find the entrances to the booths, etc.

I'm not opposed to doubling the number of booths/line-ups in the future if it's deemed necessary, but I wanted to keep the scale small for now.

CawsForConcern commented:
> > Looks fine to me, are you waiting for someone else's green light?
> 
> Trep wanted to have input on SC17, was waiting on their reply before un-drafting this

Trep gave the OK, moved to non-draft yay

Conflee commented:
YAML error rip.

Also its all unlocked and anyone can walk through by default right?

CawsForConcern commented:
> YAML error rip.
> 
> Also its all unlocked and anyone can walk through by default right?

Yes, the checkpoint starts completely unlocked. Doors open, greenlight on. It has to be explicitly closed using the buttons in the booth.

CawsForConcern committed:
Revert "- add: (Centcomm G24) Security / Customs checkpoint that can be used by admins for events"

This reverts commit 7b4b691c4310ce14eb0ca9d336f0acb367f44fea.

CawsForConcern commented:
Oops, accidentally pasted the map `.yml` file over the map prototype :V

That explains the YAML linter errors

CawsForConcern commented:
Found stuff when I was trialmining that I want to address before this gets merged

Central Command G24
- [x] No stoves on ovens in kitchen
- [x] No sink in kitchen
- [x] Cell rechargers needed (for recharging devices, especially translators)
- [x] APC just left of armory is entirely disconnected from the power grid

CawsForConcern committed:
- add: (Centcomm G24) Sink to kitchen
- add: (Centcomm G24) Adds some rechargers, useful for charging your translators and other gadgets
- fix: (Centcomm G24) Disconnected APC by Armory

CawsForConcern committed:
- add: (Centcomm SC17) More rechargers in public places and in departments
- add: (Centcomm SC17) Recharger for chemistry
- add: (Centcomm SC17) Two more mindshields in CC Debrief

CawsForConcern commented:
Included some fixes and improvements (sink for G24 kitchen, fixed a disconnected APC on G24 by Armory, etc).

Updated changelog. Should be good to merge!

CawsForConcern committed:
Merge branch 'starlight-dev' into CentCommCheckpoints
